public/lessons/graph-theory-intro/intro.mdx

```diff
@@ -0,0 +1,118 @@
+# Introduction to Graph Theory
+
+If you have ever seen a problem that involves a set of entities and establishes relationships
+between them, you have seen a problem that can potentially be solved using Graph Theory. Graphs are
+a fundamental data structure in Computer Science and can be used to model a wide variety of problem
+scenarios. Given a graphical interpretation of a problem, we can take advantage of many graph-based
+algorithms available to come up with a solution. Some noteworthy examples of classical graph theory
+problems include graph traversal, finding shortest-paths, topological sorting, and many more. This
+makes Graph Theory a fascinating and essential topic to master when it comes to Competitive
+Programming.
+
+<Graph
+  id="graph"
+  data={{
+    nodes: [
+      { id: 0 }, 
+      { id: 1 }, 
+      { id: 2 }, 
+      { id: 3 },
+      { id: 4 },
+    ],
+    edges: [
+      { source: 0, target: 1},
+      { source: 0, target: 3},
+      { source: 0, target: 4},
+      { source: 1, target: 2},
+      { source: 1, target: 3},
+      { source: 2, target: 3},
+      { source: 3, target: 4},
+    ]
+  }}
+/>
+
+## Terminology
+
+Formally, a **graph** can be defined as $G=(V,E)$ where $V$ is a set of **vertices** (or nodes) and
```

Comment:
I feel like this could be better as bulleted list for each bold term

Reply:
Yeah, I thought about that as well. I thought the page already had many bullet points. But I'll see how that looks if I get a chance.

Reply:
I'm gonna leave it as is for now since stem day is soon